### DISCUSSION TITLE
Support for FreeBSD

### DISCUSSION THREAD

**Discussion by ghost** [pyinstaller/pyinstaller#7878] [PyInstaller]
I am using https://www.freebsd.org/platforms/mips/ on a MIPS board. I want to deploy an application written in python on that board. 
How to do that?

**Comment by rokm** [comment]
Why not just install python in your FreeBSD on MIPS board, and run your application there as-is?

If you want to use PyInstaller, you will need to set up a working python environment that runs under the same OS and architecture (so FreeBSD on MIPS) anyway...
